src/components/incident/CreateAckSignOffAction.tsx

```diff
@@ -29,6 +29,7 @@ const CreateAck: React.FC<CreateAckPropsType> = ({ onSubmitAck, signOffActionPro
       },
       expiration: selectedDate && selectedDate.toISOString(),
       timestamp: new Date().toISOString(),
+      description: msg,
```

Comment:
Do I see that right that you now post the description twice? Once within event and once outside of it? I think it that case it would be better to have to different bodies: one for posting to v1 and one for posting to v2

Reply:
You will likely have two different timestamps as well.